### PR TITLE
Improve CLI functionality

### PR DESCRIPTION
**Pull request by Doralitze**
This adresses isse #130.

### CODE REVIEW

File: src/controller/cli/show_command.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
+from typing import TYPE_CHECKING
```
Comment (maxkaussow) [inline src/controller/cli/show_command.py:1]: encoding und Kommentar

File: src/controller/cli/show_command.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+if TYPE_CHECKING:
+    from controller.cli.cli_context import CLIContext
+    from argparse import ArgumentParser
+from controller.cli.command import Command
+from controller.file.read import read_document
+from controller.file.transmitting_to_fish import transmit_to_fish
+
+
+class ShowCommand(Command):
+
```
Comment (maxkaussow) [inline src/controller/cli/show_command.py:12]: doku

File: src/controller/cli/show_command.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    def __init__(self, context: "CLIContext"):
+        super().__init__(context, "showctl")
+
+    def configure_parser(self, parser):
+        subparsers = parser.add_subparsers(help="showctl commands", dest="showaction")
```
Comment (maxkaussow) [inline src/controller/cli/show_command.py:17]: doku

**Review by maxkaussow — APPROVED** [review]
passt sonst

File: src/controller/cli/show_command.py (hunk 1 above)
Comment (Doralitze) [reply src/controller/cli/show_command.py:8]: Addressed.

File: src/controller/cli/show_command.py (hunk 2 above)
Comment (Doralitze) [reply src/controller/cli/show_command.py:19]: Addressed.

File: src/controller/cli/show_command.py (hunk 3 above)
Comment (Doralitze) [reply src/controller/cli/show_command.py:34]: Addressed.

**Pull request merged**